mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -2089,7 +2089,7 @@ void GlobalOp::build(OpBuilder &builder, OperationState &result, Type type,
                      Attribute value, uint64_t alignment, unsigned addrSpace,
                      bool dsoLocal, bool threadLocal, SymbolRefAttr comdat,
                      ArrayRef<NamedAttribute> attrs,
-                     DIGlobalVariableExpressionAttr dbgExpr) {
+                     ArrayRef<Attribute> dbgExprs) {
```

Comment:
```suggestion
                     ArrayRef<DIGlobalVariableExpressionAttr> dbgExprs) {
```
Can this be typed (here and in the other places)?

Reply:
I tried this but it did not work. The `DIGlobalVariableExpressionArrayAttr` becomes an ArrayAttr and that takes an `ArrayRef<Attribute>` and compiler does not allow `ArrayRef<DIGlobalVariableExpressionAttr>` to be passed for that. I thought doing manual conversion was not worth the gain.

Reply:
Ok makes sense! I think DIGlobalVariableExpressionArrayAttr has a verifier so things should ultimately be checked.